File: C++/Iniciante/1048.cpp
#include<iostream>
#include<iomanip>
#include<cmath>
  
using namespace std;
  
int main (){
      
    double reaj, nvsal, sal;
    int porc;
    
    cout << fixed << setprecision(2);
    cin >> sal;
    
    if(sal<=400){
    	nvsal=sal+sal*15/100;
    	reaj=sal*15/100;
    	porc=15;
    }else if(sal>400 && sal<=800){
    	nvsal=sal+sal*12/100;
    	reaj=sal*12/100;
    	porc=12;
    }else if(sal>800 && sal<=1200){
    	nvsal=sal+sal*10/100;
    	reaj=sal*10/100;
    	porc=10;
    }else if(sal>1200 && sal<=2000){
    	nvsal=sal+sal*7/100;
    	reaj=sal*7/100;
    	porc=7;
    }else{
    	nvsal=sal+sal*4/100;
    	reaj=sal*4/100;
    	porc=4;
    }
    cout << "Novo salario: " << nvsal << endl;
    cout << "Reajuste ganho: " << reaj << endl;
    cout << "Em percentual: " << porc << " %" << endl;
    	
return 0;   
}

File: Python3/Iniciante/1048.py
# -*- encoding: utf-8 -*-

X = float(input())

if (X>=0 and X<=400):
    NS = X * 1.15
    R = X * 0.15
    P = 15

elif (X>400 and X<=800):
    NS = X * 1.12
    R = X * 0.12
    P = 12

elif (X>800 and X<=1200):
    NS = X * 1.10
    R = X * 0.10
    P = 10

elif (X>1200 and X<= 2000):
    NS = X * 1.07
    R = X * 0.07
    P = 7

else:
    NS = X * 1.04
    R = X * 0.04
    P = 4

print("Novo salario: %.2f" % NS)
print("Reajuste ganho: %.2f" % R)
print("Em percentual: %d %%" % P)
